worlds/sc2/rules.py

```diff
@@ -480,6 +480,41 @@ def terran_survives_rip_field(self, state: CollectionState) -> bool:
                         and self.terran_sustainable_mech_heal(state))
         )
 
+    def zerg_maw_requirement(self, state: CollectionState) -> bool:
+        """
+        Ability to cross defended gaps, deal with skytoss, and avoid costly losses.
+        :param state:
+        :return:
+        """
+        return (
+                state.has_all({item_names.NYDUS_WORM, item_names.SWARM_QUEEN}, self.player)
```

Comment:
Ventral Sacs as alternative to Nyduses?

Reply:
I'll put it in as advanced tactics; in testing, overlord drops were very, very risky. They don't last long in the rip field and it means reinforcing your army mid-fight is more difficult, as is running away from a bad encounter.

With Nydus, you basically just need a spotter. They can survive Brutal rip-field + 1 photon cannon while building, and if you happen to attack at the same time as the enemy wave arrives, you can load your units back in to fight again later.